pkg/toolkit/commands/httprunner.go

```diff
@@ -89,6 +89,9 @@ func MarshalHTTPRequest(ctx context.Context, rbd *RunBindDef, rhd *RunHTTPDef, p
 			}
 		}
 		if !bound {
+			if _, ok := params[k]; !ok {
+				continue
+			}
```

Comment:
I was getting 400 errors from the assistant, and eventually tracked it down to this inserting "null" values for fields that were missing from the input parameters.